homeassistant/components/vicare/sensor.py

```diff
@@ -228,6 +228,14 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
         unit_getter=lambda api: api.getGasSummaryConsumptionHeatingUnit(),
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
+    ViCareSensorEntityDescription(
+        key="gas_summary_consumption_heating_lastsevendays",
+        name="Heating gas consumption last seven days",
+        native_unit_of_measurement=UnitOfVolume.CUBIC_METERS,
+        value_getter=lambda api: api.getGasSummaryConsumptionHeatingLastSevenDays(),
+        unit_getter=lambda api: api.getGasSummaryConsumptionHeatingUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
```

Comment:
Is this really right? If it's a running 7 days it could actually decrease?

Reply:
Would like to point out following:

1. Summary sensor hotwater_gas_summary_consumption_heating_lastsevendays is there in vicare sensor.py with the similar setup, no issues.

    ViCareSensorEntityDescription(
        key="hotwater_gas_summary_consumption_heating_lastsevendays",
        name="Hot water gas consumption last seven days",
        native_unit_of_measurement=UnitOfVolume.CUBIC_METERS,
        value_getter=lambda api: api.getGasSummaryConsumptionDomesticHotWaterLastSevenDays(),
        unit_getter=lambda api: api.getGasSummaryConsumptionDomesticHotWaterUnit(),
        state_class=SensorStateClass.TOTAL_INCREASING,
    ),

2. Summary sensors were added here: Vicare Gas & Power consumption summary sensors (#66458) by Hans Oischinger <hans.oischinger@gmail.com> on 2022-05-09 13:11:09. Hot water sensor for last seven days was added but heating sensor for last seven days was left out by author for some reason unknown to me. Since then I have added it manually each time I have upgraded my Home Assistant instance. Works perfectly, but it is quite annoying to update the code after each upgrade.